### ISSUE TITLE
[Discussion] How I Lock My Bike in Berlin

### ISSUE THREAD

**Issue by malithmcr** [diessica/diessica.github.io#8]
> About https://diessi.ca/blog/how-i-lock-my-bike-in-berlin/

:D :D nice.. I got the same lock and one extra.

**Comment by diessica** [comment]
Oh, did it help you? ❤️  I am glad!

**Comment by malithmcr** [comment]
Yeah for now. :D my last bike got stolen with cheaper lock. hopefully, this will work.  :) :)  btw nice blog.

**Comment by diessica** [comment]
ugh, sad to hear! Even the good ones can't protect us though. Nowadays I just make sure I'm doing my best when it comes to what I have control over; mostly boils down to locking away carefully in the safest place I can.

Happy you like the blog! Thank you!

Enjoy the rides  🚲
